ori/volume/server/volume_create.go

```diff
@@ -80,36 +81,40 @@ func (s *Server) createImageFromVolume(ctx context.Context, log logr.Logger, vol
 		image.Spec.Encryption.EncryptedPassphrase = encryptedPassphrase
 	}
 
+	log.V(2).Info("Setting volume metadata to image")
 	if err := apiutils.SetObjectMetadata(image, volume.Metadata); err != nil {
 		return nil, fmt.Errorf("failed to set metadata: %w", err)
 	}
 	apiutils.SetClassLabel(image, volume.Spec.Class)
 	apiutils.SetManagerLabel(image, apiutils.VolumeManager)
 
-	log.V(2).Info("Create image in store")
+	log.V(2).Info("Creating image in store")
 	image, err = s.imageStore.Create(ctx, image)
 	if err != nil {
 		return nil, fmt.Errorf("failed to create image: %w", err)
 	}
 
+	log.V(2).Info("Image created", "imageID", image.Metadata.ID)
 	return image, nil
 }
 
 func (s *Server) CreateVolume(ctx context.Context, req *ori.CreateVolumeRequest) (res *ori.CreateVolumeResponse, retErr error) {
 	log := s.loggerFrom(ctx)
+	log.V(1).Info("Creating volume", "args", *req)
```

Comment:
Can we set the logger context to use the image ID? That way we don't have to log the complete request struct.

Reply:
Done